translator/conventions/cloudprovider/aws.go

```diff
@@ -0,0 +1,36 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cloudprovider
```

Comment:
This package does not reflect what is in this file.

Reply:
Good callout, renamed package to `vendorconventions` to leave room for `azure.go` etc in the future.

Reply:
:))) You just said that azure is not a cloud provider? :))))

Reply:
cloudprovider -> vendorconventions to leave room for azure :)) so azure is not a cloud provider just a vendor :)))

Reply:
Hahaha shhhhh :)